nano/node/rpc.cpp

```diff
@@ -391,6 +391,25 @@ void nano::rpc_handler::account_block_count ()
 	response_errors ();
 }
 
+void nano::rpc_handler::account_confirmation_height ()
```

Comment:
Probably it can be part of account_info instead?

Reply:
I did know about that :) yes makes sense to go there, thanks!